InsertData/src/InsertData.java

```diff
@@ -0,0 +1,309 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.Document;
+import org.json.JSONException;
+import org.json.simple.JSONArray;
+//import org.json.JSONObject;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.util.Arrays;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.util.JSONParseException;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class InsertData {
+		/*  public static void main(String[] args)
```

Comment:
You could either remove the commented code or add comments as to why it is commented out. This could help someone reading the code.

Reply:
Changes made.